lib/kafo/puppet_command.rb

```diff
@@ -43,15 +43,15 @@ def self.search_puppet_path(bin_name)
       File.join([bin_path, bin_name].compact)
     end
 
-    def self.is_aio_puppet?
+    def self.aio_puppet?
```

Comment:
You're changing this method name, but not the tests for it. That causes test failures. Please also change it in `test/kafo/puppet_command_test.rb`.

Reply:
This one is fixed now!

Reply:
@ekohl is this ready to get in? so I can rebase https://github.com/theforeman/kafo/pull/358 on top of this one, and make it ready.